lms/templates/dashboard/_dashboard_entitlement_unenrollment_modal.html

```diff
@@ -0,0 +1,33 @@
+<%page expression_filter="h"/>
+
+<%!
+from django.utils.translation import ugettext as _
+%>
+
+<div id="unenroll-entitlement-modal" class="entitlement-modal unenroll-entitlement-modal" aria-hidden="true">
+  <div class="entitlement-modal-inner-wrapper" role="dialog" aria-labelledby="unenroll-entitlement-modal-title">
+    <button class="close-entitlement-modal">
+      <span class="icon fa fa-remove" aria-hidden="true"></span>
+      <span class="sr">
+        ## Translators: this is a control to allow users to exit out of this modal interface (a menu or piece of UI that takes the full focus of the screen)
+        ${_("Close")}
+      </span>
+    </button>
+
+    <header class="entitlement-unenroll-header">
+      <h2 id="unenroll-entitlement-modal-title">
+        <span id='entitlement-track-info'></span>
+        <span id='entitlement-refund-info'></span>
```

Comment:
Classes here. Also, when you pass these classes to the JS through the options object, make sure to namespace them: so '.unenroll-entitlement-modal entitlement-track-info' not just 'entitlement-track-info'.

That will keep us from hitting the issue with styling/binding-events specificity that occurred earlier and is a likely fix for that issue.